tests/baselines/reference/tsxReactEmitEntities.js

```diff
@@ -12,5 +12,5 @@ declare var React: any;
 
 
 //// [file.js]
-React.createElement("div", null, "Dot goes here: · &notAnEntity; ");
+React.createElement("div", null, "Dot goes here: \u00B7 &notAnEntity; ");

```

Comment:
This is because the synthetic literal now passes through `escapeNonAsciiCharacters` which encodes the dot.